integration/integration_test.py

```diff
@@ -0,0 +1,49 @@
+"""@package integration_test.py
```

Comment:
One high-level question on style nits; we have a bunch of new Python in this PR and it's going to grow going forwards. Should we have a CI Python format check? See https://github.com/envoyproxy/envoy/blob/master/tools/format_python_tools.sh for what Envoy has